raft/src/main/java/org/apache/kafka/raft/RaftRequest.java

```diff
@@ -45,11 +45,14 @@ public long createdTimeMs() {
         return createdTimeMs;
     }
 
+
     public static class Inbound extends RaftRequest {
         public final CompletableFuture<RaftResponse.Outbound> completion = new CompletableFuture<>();
+        public final short apiVersion;
 
-        public Inbound(int correlationId, ApiMessage data, long createdTimeMs) {
+        public Inbound(int correlationId, ApiMessage data, long createdTimeMs, short apiVertion) {
```

Comment:
Same question here.

Reply:
Used to store the API version of the FetchRequest.